budgetportal/webflow/serializers.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework.serializers import ModelSerializer
+
+from budgetportal.models import ProvInfraProject
+
+
+class ProvInfraProjectSerializer(ModelSerializer):
```

Comment:
I think you want to be placing the URL field in this class

Reply:
url field added

https://github.com/vulekamali/datamanager/pull/190/commits/75b5d617d4a9818a4b151f253ca4476a0d7beb6b